manual-provision-payg/testsuite/start.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# start.sh paygo provision.
+echo "Provisioning paygo..."
+sleep 2
+echo "Please wait."
+#mount unmounted removable devices
+udisksctl dump |
+awk -F':\n' -v'RS=\n\n' '/[ \t]*HintAuto:[ \t]*true/&&/\.Filesystem:/{
+                             print $1
+}' |
+while read dev
+do
+    udisksctl mount --object-path "${dev##*/UDisks2/}"
+done
+
+lsblk
+device_id=$(python3.5 /var/eos-factory-test/code_gen.py)
```

Comment:
This should be `python3` rather than `python3.5`. Master images these days have `python3.7` so otherwise there's a fatal error "python3.5: command not found"

Reply:
Changed to python3